README.md

```diff
@@ -48,10 +48,12 @@ configuration necessary.
   :resources {
 
     ; The resources type indicates where lein-ezbake gets resources from.
-    ; Currently only resources stored in the lein-ezbake jar can be used. Future
-    ; versions of lein-ezbake may support pulling these resources instead from a
+    ; Resources stored in the lein-ezbake jar or in a specified directory can be used.
```

Comment:
This wording doesn't quite reflect the behavior.  Something more like:

> Resources can be copied from jars composed by `lein-ezbake` as well as optionally a directory specified by the `include-dir` option.  This is useful where a project itself needs to specify resources. When specified, the `include-dir` resources will replace files from the included jars of the same name.

Reply:
Updated